test/api/v3/unit/libs/email.test.js

```diff
@@ -137,6 +155,29 @@ describe('emails', () => {
     });
   });
 
+  describe('getGroupUrl', () => {
+    it('returns correct url if group is the tavern', () => {
+      let attachEmail = requireAgain(pathToEmailLib);
```

Comment:
requireAgain shouldn't be needed because you're not doing anything that necessitates clearing the require cache.